File: articles/contestability-appeals-and-algorithmic-due-process/typescript/contestability_score.ts
const notice = 0.70;
const reasons = 0.62;
const evidenceAccess = 0.48;
const humanReview = 0.55;
const correction = 0.52;
const remedy = 0.44;
const score = (notice + reasons + evidenceAccess + humanReview + correction + remedy) / 6.0;
console.log(`contestability_score=${score.toFixed(4)}`);

File: articles/contestability-appeals-and-algorithmic-due-process/java/ContestabilityScore.java
public class ContestabilityScore {
    public static void main(String[] args) {
        double notice = 0.70;
        double reasons = 0.62;
        double evidenceAccess = 0.48;
        double humanReview = 0.55;
        double correction = 0.52;
        double remedy = 0.44;
        double score = (notice + reasons + evidenceAccess + humanReview + correction + remedy) / 6.0;
        System.out.printf("contestability_score=%.4f%n", score);
    }
}
